managed-ledger/src/test/java/org/apache/bookkeeper/mledger/impl/ManagedCursorTest.java

```diff
@@ -4874,8 +4873,9 @@ public void asyncOpenLedger(final long lId, final DigestType digestType, final b
                 final OpenCallback cb, final Object ctx) {
             if (ledgerErrors.containsKey(lId)) {
                 cb.openComplete(ledgerErrors.get(lId), null, ctx);
+            }else{
```

Comment:
please use the default formatting code style that we use in Pulsar code base, [instructions for setting it up in IntelliJ](https://pulsar.apache.org/contribute/setup-ide/#configure-code-style).